main/zoe/guide/contracts/loan.md

```diff
@@ -0,0 +1,143 @@
+by the Maintenance Margin Requirement (`mmr`) in the terms of the
+contract.
+
+The loan does not have a distinct end time. Rather, if the
+value of the collateral changes such that insufficient margin is
+provided, the collateral is liquidated, and the loan is closed. At any
+time, the borrower can add collateral or repay the loan with interest,
+closing the loan. The borrower can set up their own margin calls by
+getting the [`priceAuthority`](/zoe/guide/price-authority.md) from the terms and calling
+`E(priceAuthority).quoteWhenLT(allCollateral, x)` where x is the value
+of the collateral in the Loan brand at which they want a reminder to
+addCollateral.
+
+Note that all collateral must be of the same brand and all of the
+loaned amount and interest must be of the same (separate) brand.
+
+## Terms
+* `mmr` (default = 150) - the Maintenance Margin Requirement, in
+   percent. The default is 150, meaning that collateral should be
+   worth at least 150% of the loan. If the value of the collateral
+   drops below `mmr`, liquidation occurs.
+* [`priceAuthority`](/zoe/guide/price-authority.md) - will be used for getting the current value of
+   collateral and setting liquidation triggers.
+* `autoswapInstance` - The running contract instance for an
+   [Autoswap](./autoswap.md) or [Multipool
+   Autoswap](./multipoolAutoswap.md) installation. The `publicFacet`
+   of the instance is used for producing an invitation to sell the
+   collateral on liquidation.
+* `periodAsyncIterable` - the [asyncIterable](https://javascript.info/async-iterators-generators) used for notifications
+   that a period has passed, on which compound interest will be
+   calculated using the `interestRate`.
+* `interestRate` - the rate in [basis points](https://www.investopedia.com/terms/b/basispoint.asp) that will be multiplied
+   with the debt on every period to compound interest.
+
+## IssuerKeywordRecord
+
+The following is used for all keyword records, regardless of role in
+the contract:
+
+* Keyword: 'Collateral' - The issuer/payment for the digital assets to be
+   escrowed as collateral.
+* Keyword: 'Loan' - The issuer/payment for the digital assets to be loaned
+   out.
+
+## The Lender
+
+The lender puts up the amount to be loaned to the borrower, but has no
+further actions. The loan is ongoing until it is paid back entirely or
+liquidated, at which point the lender receives a payout. This means
+that the payout for the lender will be in Loan-branded digital assets,
+not Collateral-brand. (The only exception to this is if the scheduling
+of liquidation triggers with the `priceAuthority` results in a error. In
+that case, we have no choice but to give the collateral to the lender.
+The borrower has already exited with their loan.)
+
+The lender will want the interest earned from the loan + their refund
+or the results of the liquidation. If the price of collateral drops
+before we get the chance to liquidate, the total payout could be zero.
```

Comment:
```suggestion
The lender will want the loan interest in addition to either their refund
or the liquidation results. If the collateral price drops
before we get the chance to liquidate, the total payout could be zero.
```

Reply:
In the above, instead of "we get the chance to liquidate", should it be "they get the chance to liquidate"? Not quite sure who's doing the liquidating here.

Reply:
the contract is doing the liquidating